libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/marker/GeocodingComponent.kt

```diff
@@ -27,6 +28,7 @@ internal class GeocodingComponent(
 
         val geocoder = Geocoder.create(accessToken)
         destinationViewModel.state
+            .filter { it.destination?.features == null }
```

Comment:
Nice simple fix. Integrating search will make us want to actually add a cache.

But right now, it's very unlikely you long press at the same place :). It may be worth adding a cache before integrating search. But 🚀  with this